src/i18n/es/ui.ts

```diff
@@ -59,4 +59,6 @@ export default UIDictionary({
 	'404.title': 'No encontrado',
 	'404.content': 'Esta página no está en nuestro sistema solar.',
 	'404.linkText': 'Volver a la página principal.',
+	// BaaS Guides vocabulary
+	'baas.navTitle': 'Más guías de backend como servicio',
```

Comment:
These translations are from native speakers or by some tool?  We generally don't include other language ui labels in new content PRs, as we need the agreement of, and to be following the patterns made by the community members who maintain these. 

The translation doesn't sound wrong, but I'd personally keep only English content changes to this PR so we can give our community translators the chance to make an appropriate translation according to what they think works best.